uberon_edit.obo

```diff
@@ -19,6 +19,7 @@ subsetdef: non_informative "abstract class brought in to group ontology classes
 subsetdef: organ_slim "organs, excluding individual muscles and skeletal elements"
 subsetdef: pheno_slim "Phenotype slim"
 subsetdef: phenotype_rcn "A subset specifically created for the 2012 Phenotype RCN meeting. Includes some human-specific terms that may eventually be removed when they are adequately represented with part of relationships in FMA"
+subsetdef: UBERON:8410007 "Classes tagged with this subset property were added upon request from the Human Cell Atlas (HCA)."
```

Comment:
this doesn't seem to be formatted like the other subset defs - will UBERON:841007 become the label? Is this what you intended?

It's not letting me open this branch, so I can't look in Protege.

Reply:
@nicolevasilevsky 
> this doesn't seem to be formatted like the other subset defs - will UBERON:841007 become the label? Is this what you intended?
> It's not letting me open this branch, so I can't look in Protege.

Thanks Nicole - no, I meant "added_for_HCA" to be the label... let me try again!

Reply:
@nicolevasilevsky 
I tried again and see the same issue. I'll attach a screenshot of what it looks like in Protege for me:
![Screen Shot 2021-01-13 at 15 31 07](https://user-images.githubusercontent.com/4982516/104465539-6d3b2900-55b4-11eb-84e3-25853286e48e.png)